packages/flutter_migrate/README.md

```diff
@@ -0,0 +1,46 @@
+### Flutter Migrate
+
+## Overview
+
+This is a tool that helps migrate legacy Flutter projects generated with old version of
+Flutter to modern Flutter templates. This allows old apps to access new features, update
+key dependenices and prevent slow bitrot of projects over time without domain knowledge
+of individual platforms like Android and iOS.
+
+## Prerequisites
+
+This tool supports migrating apps generated with Flutter 1.0.0 and newer. However, projects
+generated with older versions of Flutter (beta, alpha, etc) may still be compatible with
+this tool, but results may vary and official support will not be provided.
+
+Projects that contain heavy modifications to the project's platform directories (eg,
+`android/`, `ios/`, `linux/`) may result in many conflicts.
+
+Currently, only full Flutter apps are supported. This tool will not work properly with
+plugins, add-to-app, or non-flutter packages.
```

Comment:
nit: Flutter

Reply:
Although, if it only works with apps, why specify "non-Flutter" packages? It doesn't work with any packages, right?

Reply:
I'll just remove reference to non-flutter-packages. This is just a brief starter readme that I plan on expanding as more of this gets landed. Expect updates to this that makes it much more comprehensive.